pyi.py

```diff
@@ -438,14 +437,14 @@ def run(self, tree: ast.AST) -> Iterable[Error]:
         yield from self.errors
 
 
-@attr.s
+@dataclass
 class PyiTreeChecker:
-    name = "flake8-pyi"
-    version = __version__
+    name: ClassVar[str] = "flake8-pyi"
+    version: ClassVar[str] = __version__
 
-    tree = attr.ib(default=None)
-    filename = attr.ib(default="(none)")
-    options = attr.ib(default=None)
+    tree: Optional[ast.Module] = None
+    filename: str = "(none)"
+    options: Any = None
```

Comment:
I... wasn't actually sure what the proper type annotation for this was

Reply:
I think `Any` is fine for now, but if you want, you can add `print(self.options)` somewhere and run the tests.

Reply:
Seems like it's an `argparse.Namespace`.